hier_config/host.py

```diff
@@ -15,222 +18,146 @@ class Host:
         import yaml
         from hier_config.host import Host
 
-        options = yaml.load(open('./tests/files/test_options_ios.yml'))
+        options = yaml.load(open('./tests/fixtures/options_ios.yml'), loader=yaml.SafeLoader())
```

Comment:
What is we had a 'options_file' option that called a method to load the file in yaml and set the returned output to self.hconfig_options?

Reply:
I tried this but ran into typing issues. I can re-attempt at some point in the future.